### DISCUSSION TITLE
[suggestion] Gameplay and Future :D

### DISCUSSION THREAD

**Discussion by Uncle-Ulty** [tsunamayo/Starship-EVO#3980] [Suggestions]
I'm seeing several requests and good suggestions for the game. It's a fact that there is a lot of stuff that can be done to improve the game and make it more interesting, as with any game, and there will always be something to do to improve the game. We can see this in other old games that are online till current days, such as EVE, WoW, and so on, they still receive updates, expansions, quality of life and etc. However, I remember a piece of wise advice from a project that I was working on "The good is the enemy of optimal", and "you will be always fixing and improve your project".

What I wanna say is, there a lot of things to do, but there is an objective in mind: the game. I'm seeing several good suggestions, concerning building, that could be implemented at any point of the road, with or without full gameplay. So, I think the main suggestion that everybody should make is: focus on finishing the whole gameplay cycle, to have a functional game, with survival, craft, building, and whatever. A fully functional game would attract more players and more income, which would allow to hire of staff members and accelerate the development process. But please, don't interpret this as an arrogant comment of a "wanna teach" or "wanna demand" person. I just think that would be wise if the entire community agreed that with the current Novox system we already have a lot of freedom to work with, and what we really need is the gameplay. To be honest, I'm kinda lost about the new possibilities in the experimental mode, and I realized that I need to update every building I made till now! Plus, with future gameplay features, it's almost certain that we gonna receive new blocks with new features concerning the gameplay itself, e.g. item storage and fuel, that will force us to modify old buildings to make them functional. Thus, it's wise that the gameplay advances before we add new aesthetic resources. Please, don't be mad at me, it's just an opinion.

**Comment by tsunamayo** [comment]
Yeah I know and agree mate, now please go on and write this in all suggestions out there of the like we need this block or this building feature...
